message.xml

```diff
@@ -9,5 +9,8 @@ this repository contains the full copyright notices and license terms. -->
       <record model="ir.message" id="fiscalyear_end_date">
           <field name="text">The final posting date does not match the fiscal year "%(fiscalyear)s".</field>
       </record>
+      <record model="ir.message" id="timeout_exception">
+          <field name="text">Report computation timed out. You may consider increasing the timeout of the account reports.</field>
```

Comment:
We should suggest using Excel output, not increasing the timeout.

Reply:
IMHO, both options...